shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -1082,6 +1082,12 @@ - (void)flutterTextInputView:(FlutterTextInputView*)textInputView
                               arguments:@[ @(client), @(start), @(end) ]];
 }
 
+- (void)flutterTextInputView:(FlutterTextInputView*)textInputView
+    willDismissEditMenuWithTextInputClient:(int)client {
+  [_platformChannel.get() invokeMethod:@"System.onDismissSystemContextMenu"
```

Comment:
Should this be `ContextMenuonDismissSystemContextMenu` in order to align with `ContextMenu.showSystemContextMenu` etc.? Or is this the typical naming pattern?

Reply:
Renamed to `ContextMenu.onDismissSystemContextMenu`